sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/models/KeyRotationPolicy.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Represents the rotation policy for a key.
+ */
+@Fluent
+public final class KeyRotationPolicy {
+    @JsonProperty(value = "id", access = JsonProperty.Access.WRITE_ONLY)
+    private String id;
+
+    @JsonProperty(value = "lifetimeActions")
+    private List<LifetimeAction> lifetimeActions;
+
+    @JsonProperty(value = "attributes")
+    private KeyRotationPolicyAttributes attributes;
+
+    /**
+     * Get the {@link KeyRotationPolicy policy} id.
+     *
+     * @return The {@link KeyRotationPolicy policy} id.
+     */
+    public String getId() {
+        return this.id;
+    }
+
+    /**
+     * Get the actions that will be performed by Key Vault over the lifetime of a key. At the moment,
+     * {@link LifetimeAction} can only have two items at maximum: one for rotate, one for notify. The notification time
```

Comment:
Unless this is internal, I would leave out the `At the moment...` as well as `The notification time...` since that might change without an accompanying code change

Reply:
This class is internal, but I will make sure no public classes contain this wording.

Reply:
Would we not have any mention if this in our documentation even after the service goes GA? I can understand they might change the values but I'm not sure how many people actually would bother to go into Microsoft Docs and check :P

Reply:
Totally valid point... my concern is that this will change at some point at the service side and this doc-comment will become stale. We'll probably never go back and change until a customer notices it. From my testing key rotation returns really good error messages for these cases (but let me know if you find that not to be the case because we'll want to improve that).

All that said, this is just a suggestion - feel free to ignore it because it can definitely go either way.